Source/Core/ScreenSpaceEventHandler.js

```diff
@@ -768,5 +776,12 @@ define([
         return destroyObject(this);
     };
 
+    /**
+     * The amount of time, in milliseconds, that mouse events will be disabled after
+     * receiving any touch events, such that any emulated mouse events will be ignored.
+     * @default 800
+     */
+    ScreenSpaceEventHandler.MouseEmulationIgnoreMilliseconds = 800;
```

Comment:
Doesn't particularly matter to me.  I think camel case is fine.  The camera example was just the only other place I could think of that we do this.